globalvariable/global.go

```diff
@@ -0,0 +1,8 @@
+package globalvariable
+
+const (
+	Strbase = 10
```

Comment:
I suggest reading the `golang` naming convention from 
- https://github.com/golang/go/wiki/CodeReviewComments
- https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md
- https://golang.org/doc/effective_go.html#functions
- https://smartystreets.com/blog/2018/10/go-naming-tutorial/